### DISCUSSION TITLE
Lesson 1: Question Related to Mining

### DISCUSSION THREAD

**Discussion by Ifra001** [Cyfrin/foundry-full-course-cu#1749] [General]
Patrick said that the block time is proportional to how hard the mining problem is and that the system can change how hard or how easy the Mining problem is. Who controls how hard the mining problem should be bcz i am a bit confused😅

**Comment by Niferu** [comment]
Hi @DontMind-me 

The difficulty of the mining problem in a blockchain system, such as Ethereum, is not controlled by any single entity. Instead, it is an automatic process governed by the protocol of the blockchain itself. Mining difficulty is adjusted every single block on Ethereum while for example for Bitcoin it occurs every 2016 blocks (2 weeks)

**Reply by EngrPips** [reply]
Awesome explanation. Do you get it now, @DontMind-me?

**Reply by Ifra001** [reply]
Thank you so much, Now it makes so much more sense👍👍
